### PR TITLE
Add devchat.tv

### PR DESCRIPTION
**Pull request by realdubb**

### CODE REVIEW

File: code/js/inject/devchat_inject.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
+(function() {
+  "use strict";
+
+  var sk_log = require("../modules/SKLog.js");
+
```
Comment (berrberr) [inline code/js/inject/devchat_inject.js]: remove extra newline

File: code/js/inject/devchat_inject.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+  var audio = document.getElementsByTagName("audio")[0];
+
+  if(audio === undefined)
```
Comment (berrberr) [inline code/js/inject/devchat_inject.js]: To keep with the style used elsewhere (eg: https://github.com/berrberr/streamkeys/blob/master/code/js/modules/BaseController.js#L199), please write as:

```
if(audio === undefined) return null;
```

File: code/js/inject/devchat_inject.js [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+        return null;
+
+
```
Comment (berrberr) [inline code/js/inject/devchat_inject.js]: remove extra newline

File: code/js/inject/devchat_inject.js [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+  document.addEventListener("streamkeys-cmd", function(e) {
+
```
Comment (berrberr) [inline code/js/inject/devchat_inject.js]: remove extra newline

File: code/manifest.json [hunk 5]
```diff
@@ -75,4 +76,4 @@
       "description": "Toggle dislike"
     }
   }
-}
+}
```
Comment (berrberr) [inline code/manifest.json]: Add newline back in please

**Comment by berrberr** [comment]
Fix the minor style comments I had and this looks good, thanks!

BTW I enjoyed your use of `playState: ".sk-not"` to get around play state detection. Interesting use case that I hadn't thought of when I wrote it! 👍

**Comment by realdubb** [comment]
Thanks for the feedback and for making this plugin. Style suggestions fixed.

**Comment by berrberr** [comment]
Great, thanks!

**Pull request merged**